### ISSUE TITLE
Bug: `tv()` / `cva()`: array values in base and variants are not scanned

### ISSUE THREAD

**Issue by ds-pankov** [sergioazoc/oxlint-tailwindcss#25]
The `tv()` / `cva()` extractor walks **string** values but skips **arrays of strings** —
a common multi-line pattern in both libraries' docs.

### Versions
`oxlint` 1.66.0 · `oxlint-tailwindcss` 0.8.0

### Repro

`.oxlintrc.json`:
```json
{
  "jsPlugins": ["oxlint-tailwindcss"],
  "settings": { "tailwindcss": { "entryPoint": "src/styles.css" } },
  "rules": { "tailwindcss/no-unknown-classes": "error" }
}
```

```ts
import { tv } from 'tailwind-variants';
  
tv({ base: 'flex bad-class' });                       // ✅ flagged
tv({ base: ['flex bad-class'] });                     // ❌ skipped
tv({ base: ['flex', 'bad-class'] });                  // ❌ skipped
tv({ variants: { c: { p: 'bad-class' } } });          // ✅ flagged
tv({ variants: { c: { p: ['bad-class'] } } });        // ❌ skipped
```

**Expected**

All five should be reported. The array form is the idiomatic way to write
multi-line class lists in tv() and cva().

Likely also affects compoundVariants[].class, slots, compoundSlots[].class
(same shape).

**Comment by sergioazoc** [comment]
Fixed — thanks @ds-pankov, the five-line repro made this trivial to pin down.

The class extractor walked strings, template literals, ternaries, logical expressions and objects, but never recursed into arrays, so every array form you listed was silently skipped. `extractFromExpression` now handles `ArrayExpression` by recursing into each element (skipping holes and spreads), which covers all the cases at once — `base: ['flex']`, `base: ['flex', 'bad-class']`, array variant/slot values, and `cn(['flex', 'bad-class'])`. `compoundVariants` / `compoundSlots` keep their existing array-of-objects handling.

Shipping in v1.0.0 (releasing shortly). Closing as fixed — feel free to reopen if any array shape still slips through.
